core/src/wallet/ripple/explorers/RippleLikeBlockchainExplorer.h

```diff
@@ -62,25 +62,14 @@ namespace ledger {
             std::string sender;
             Option<Block> block;
             uint64_t confirmations;
+            Option<uint64_t> destinationTag;
+
             std::vector<api::RippleLikeMemo> memos;
 
             RippleLikeBlockchainExplorerTransaction() {
                 confirmations = 0;
             }
 
-            RippleLikeBlockchainExplorerTransaction(const RippleLikeBlockchainExplorerTransaction &cpy) {
```

Comment:
Why has this disappeared?

Reply:
I was pairing with @pollastri-pierre, and he thinks that we don't need this copy constructor.

Reply:
Alright then. :)